src/guest_memory.rs

```diff
@@ -394,6 +394,25 @@ impl<M: GuestMemory> GuestAddressSpace for Arc<M> {
     }
 }
 
+/// Lifetime generic associated iterator. Usually this is just `Self`.  The actual
```

Comment:
This is the description for `type I`, not for `GuestMemoryIterator`.

Reply:
Whoops.